### PR TITLE
F405 -  variable/function/... may be undefined, or defined from star …

### PR DESCRIPTION
**Pull request by guedou**
This PR from the #1277 fixes star imports.

### CODE REVIEW

**Comment by guedou** [comment]
@gpotter2 @p-l- ready to be reviewed. Please do not merge something else before this PR =)

File: scapy/arch/windows/__init__.py [hunk 1]
```diff
@@ -64,7 +67,7 @@
     socket.IPPROTO_GRE = 47
 
 from scapy.arch import pcapdnet
-from scapy.arch.pcapdnet import *
+from scapy.arc.pcapdnet import NPCAP_PATH, get_if_raw_addr, get_if_list
```
Comment (gpotter2) [inline scapy/arch/windows/__init__.py]: Typo

File: scapy/autorun.py [hunk 2]
```diff
@@ -79,7 +79,7 @@ def autorun_commands(cmds, my_globals=None, ignore_globals=None, verb=0):
             pass
     finally:
         conf.verb = sv
-    return _
+    return _  # noqa: F821
```
Comment (gpotter2) [inline scapy/autorun.py]: I doubt this will do anything :/

Comment (guedou) [reply scapy/autorun.py]: Changing this line to something else broke UTScapy. I did not manage to find a better fix ..

**Pull request merged**